.drone.yml

```diff
@@ -17,7 +17,7 @@ publish:
     username: $$DOCKER_USER
     password: $$DOCKER_PASS
     email: $$DOCKER_EMAIL
-    repo: plugins/drone-s3-sync
+    repo: ribase/drone-s3-sync
```

Comment:
No worries. I'm just pointing out things I see that would keep it from getting merged.